docs/docs/overview/quickstart/get-started-with-nextjs.md

```diff
@@ -0,0 +1,319 @@
+---
+sidebar_position: 5
+sidebar_label: Get started with NextJS
+---
+
+# NextJS Quickstart
+
+This quick start guide is written with the intention of helping you integrate Novu into your NextJS app in no time. Novu is an open-source notifications infrastructure that you can use in your app to deliver rich notifications across channels like sms, email, in-app, push, chat, etc. In this quick start, you’ll learn how to integrate email notifications into your NextJS app.
+
+You can find the code for this app [here](https://github.com/novuhq/nextjs-quickstart/)
+
+## Prerequisites
+
+- Node.js installed on your development machine (to create a NextJS app).
+- A Novu account. If you don't have one, sign up for free [here](https://web.novu.co/workflows).
+
+## Create a NextJS app to get started
+
+The first step here would be to create a NextJS app. We’ll later play with Novu in this app itself. To get started, open your terminal and create a NextJS app using the following command:
```

Comment:
We’ll later play with Novu in this app itself => We will play with Novu in this app later.